src/client/fontengine.cpp

```diff
@@ -90,7 +90,7 @@ void FontEngine::cleanCache()
 }
 
 /******************************************************************************/
-irr::gui::IGUIFont *FontEngine::getFont(FontSpec spec)
+irr::gui::IGUIFont *FontEngine::getFont(const FontSpec &spec)
```

Comment:
`FontSpec` is only a few bytes big. Taking a reference to it is actually a worsening.

Reply:
@SmallJoker, fix it https://github.com/minetest/minetest/pull/12704/commits/7b18f1da0cf9bc70b334d51fc03ca93000e70811